docs/reference/migration/migrate_6_0/mapping.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[breaking_60_mapping_changes]]
+=== Mapping changes
+
+=== Analysis
+
+==== Synonym Token Filter
+
+In 6.0, Synonym Token Filter tokenize synonyms with whatever
+tokenizer and token filters appear before it in the chain.
+
+`tokenizer` and `ignore_case` are deprecated.
+These parameters are still left for backwards compatibility
+for indices that created before 6.0.
+And elasticsearch ignores these properties for new indices.
+
```

Comment:
you should have modified `docs/reference/migration/migrate_6_0/mappings.asciidoc` (note the `s` at the end of `mapping`) instead of creating a new file?